example/src/main/java/com/l4digital/fastscroll/example/ExampleActivity.java

```diff
@@ -41,6 +43,17 @@ protected void onCreate(Bundle savedInstanceState) {
         FastScrollRecyclerView recyclerView = (FastScrollRecyclerView) findViewById(R.id.recycler_view);
         recyclerView.setLayoutManager(new LinearLayoutManager(this));
         recyclerView.setAdapter(new ExampleAdapter());
+        recyclerView.setOnFastScrollStateChangeListener(new OnFastScrollStateChangeListener() {
```

Comment:
Can you remove the state change listener from the Example? The example should be kept minimal to demonstrate the simplest implementation of the library.